### DISCUSSION TITLE
Betrieb Zendure Hub 2000 an Hoymiles 1600-4T. Nulleinspesisung über OpenDTU OB oder über Zendure App?

### DISCUSSION THREAD

**Discussion by Josef1966** [hoylabs/OpenDTU-OnBattery#1338] [General]
Ich habe ein BKW mit Hoymiles HMS-1600-4T Wechselrichter, 4 Solarplatten Trina Vertex S+ 425Wp, Shelly Pro 3EM und OpenDTU-onBattery mit Nulleinspeisung. Jetzt habe ich mir noch einen Zendure-Speicher gekauft (Solar Flow, Hub 2000 + AB2000), den ich noch in die Anlage integrieren möchte.
Wie realisiere ich dann am besten die Nulleinspeisung, über OpenDTU-onBattery am Wechselrichter oder über den CT-Modus beim Hub?
Im Anhang die von mir geplante Anordnung der einzelnen Komponenten. Ich möchte die 4 Eingänge am WR nutzen, oder macht die Belegung von nur 2 Eingängen für die Regelung über OpenDTU mehr Sinn?

![Speicher_Hoymiles_Zendure_WR-4-Eingänge](https://github.com/user-attachments/assets/85183f43-2943-4bc3-9ff2-674d264ccd6b)

**Comment by vaterlangen** [comment]
ZeroFeed über den WR ist vermutlich schneller - und laut Skizze hängt bei dir auch nichts anderes am WR. Einfach outputlimit am HUB 2000 hochdrehen und die Limitierung dem DPL überlassen.

Wenn du den Batteriestatus in OpenDTUoB sehen willst wäre #1221 evtl was für dich.
